fakestorage/object.go

```diff
@@ -686,7 +687,7 @@ func (s *Server) handleRange(obj Object, r *http.Request) (ranged bool, start in
 				if end > int64(len(obj.Content)) {
 					end = int64(len(obj.Content))
 				}
-				return true, start, end, obj.Content[start:end]
+				return true, start, end - 1, obj.Content[start:end]
```

Comment:
Shouldn't this be:
```
return true, start, end - 1, obj.Content[start:end-1]
```
?

Reply:
Nope, in the slice expression the upper bound is non-inclusive: https://go.dev/ref/spec#Slice_expressions